sample.env

```diff
@@ -67,7 +67,7 @@ SB_RALLY_POINT_LOCAL_PORT=14098
 # two options, storing to the filesystem and storing to the DigitalOcean Spaces.
 SB_FILE_STORE={"filesystem":{"path":"server/uploaded_files"}}
 # Example DigitalOcean Spaces configuration
-#SB_FILE_STORE={"doSpaces":{"endpoint":"region.digitaloceanspaces.com","accessKeyId":"ACCESS_KEY_ID","secretAccessKey":"SUPER_SECRET_ACCESS_KEY","bucket":"shieldbattery","cdnHost":"cdn.example.org"}}
+#SB_FILE_STORE={"doSpaces":{"endpoint":"https://region.digitaloceanspaces.com","accessKeyId":"ACCESS_KEY_ID","secretAccessKey":"SUPER_SECRET_ACCESS_KEY","bucket":"shieldbattery","cdnHost":"cdn.example.org"}}
```

Comment:
Please check if the provided endpoint has a protocol and if not, add `https://` to the front of it. I don't want to have to remember to update all the existing configs we have to add this